menas/ui/service/RunService.js

```diff
@@ -77,7 +77,7 @@ var RunService = new function () {
   };
 
   this.getLatestRunForLatestVersion = function (oControl, oTable, datasetName) {
-    Functions.ajax("api/runs/" + encodeURI(datasetName) + "/latest", "GET", {},
+    Functions.ajax("api/runs/" + encodeURI(datasetName) + "/latestrun", "GET", {},
```

Comment:
That endpoint doesn't exist, it was set up, but never implemented. Nobody noticed or used it, lol. It will be entirely removed with the new grouped runs navigation.